addons/web/static/src/core/errors/error_service.js

```diff
@@ -64,7 +64,13 @@ function combineErrorNames(uncaughtError, originalError) {
 async function completeUncaughtError(env, uncaughtError, originalError) {
     uncaughtError.name = combineErrorNames(uncaughtError, originalError);
     if (env.debug.includes("assets")) {
-        uncaughtError.traceback = await annotateTraceback(originalError);
+        try {
+            uncaughtError.traceback = await annotateTraceback(originalError);
+        } catch (e) {
+            // This can crash if the originalError has no stack/stacktrace property
+            console.warn("The following error could not be annotated:", originalError, e);
+            uncaughtError.traceback = formatTraceback(originalError);
```

Comment:
are you sure about this? Michael did [this](1331165c0f2a164ec0b4fdb6db72c303fddfc938) recently

Reply:
Yep, the case that triggers the problem Michael fixed (stackless error) is broken when you're in debug=assets, the same error condition that was causing an issue in formatTraceback causes a crash in annotateTraceback, but annotateTraceback was not fixed.

Reply:
(maybe I should have fixed it inside annotateTraceback though)

Reply:
> (maybe I should have fixed it inside annotateTraceback though)

That's what I was thinking indeed